vm-setup/roles/v1aX_integration_test/tasks/node_reuse.yml

```diff
@@ -110,8 +59,15 @@
       definition:
         spec:
           nodeReuse: true
+          template:
+            spec:
+              image:
+                checksum: "http://172.22.0.1/images/{{ UPGRADED_RAW_IMAGE_NAME }}.{{ IMAGE_CHECKSUM_TYPE }}sum"
+                checksumType: "{{IMAGE_CHECKSUM_TYPE}}"
+                format: "{{IMAGE_FORMAT}}"
+                url: "http://172.22.0.1/images/{{ UPGRADED_RAW_IMAGE_NAME }}"
```

Comment:
dataTemplate is not necessary?

Reply:
no, it uses same dt since we are not generating a new m3mt